README.md

```diff
@@ -69,11 +69,11 @@ end up using one of these for the lexer at one point.
 
 | prerequisite |      | version                                                                | how to install                  |
 | ------------ | ---- | :--------------------------------------------------------------------- | ------------------------------- |
-| Opam         | `>=` | [`2.0.1`](https://github.com/ocaml/opam/releases/tag/2.0.1)            | manually or via package manager |
+| OPAM         | `>=` | [`2.0.1`](https://github.com/ocaml/opam/releases/tag/2.0.1)            | manually or via package manager |
```

Comment:
should the version documented here be changed?

Reply:
it certainly should!